guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -116,7 +116,8 @@ Ensure {SmartProxy} has access to `{RepoRHEL7ServerSatelliteMaintenanceProductVe
 +
 [options="nowrap" subs="attributes"]
 ----
-# {foreman-maintain} self-upgrade
+# subscription-manager repos --enable rhel-7-server-satellite-maintenance-6.11-rpms
```

Comment:
If you do this, please use an attribute:

```suggestion
# subscription-manager repos --enable {RepoRHEL7ServerSatelliteMaintenanceProductVersion}
```

Having said that: I thought Capsules get their content from Satellite. In step 3.2 the repo should already be synced so I wonder why this is needed in the first place. However, I'm not that up to date on capsules and their content so perhaps I'm wrong.

Reply:
>Having said that: I thought Capsules get their content from Satellite. In step 3.2 the repo should already be synced so I wonder why this is needed in the first place. However, I'm not that up to date on capsules and their content so perhaps I'm wrong.

It's been confirmed by relevant teams because this change was in priority.